RPC tests fail on Windows with local TimeZone UTC+2

<!-- 
Have you searched for similar issues before posting it?

If you have discovered a bug in the bot, please [search the issue tracker](https://github.com/freqtrade/freqtrade/issues?q=is%3Aissue). 
If it hasn't been reported, please create a new issue.

Please do not use bug reports to request new features.
-->

## Describe your environment

  * Operating system: Windows 11
  * Python Version: Python 3.10.9 (`python -V`)
  * CCXT version: 2.8.17 (`pip freeze | grep ccxt`)
  * Freqtrade Version: freqtrade 2023.3.dev-44b100507 
 

## Describe the problem:

Two tests related to RPC fail on Windows where local timezone is set to UTC +X, where X>=2

### Steps to reproduce:

  1. Run tests on Windows 11 with TimeZone UTC +X, where X>=2
  
### Observed Results:

  * Tests fail

### Relevant code exceptions or logs

Note: Please copy/paste text of the messages, no screenshots of logs please.

`rpc.test_rpc.test_rpc_health` with next log:
```python
tests\rpc\test_rpc_apiserver.py:1796 (test_health)
botclient = (<freqtrade.freqtradebot.FreqtradeBot object at 0x000001E75CA2F9A0>, <starlette.testclient.TestClient object at 0x000001E75CC9ACB0>)

    def test_health(botclient):
        ftbot, client = botclient
    
>       rc = client_get(client, f"{BASE_URI}/health")

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = tzlocal(), dt = datetime.datetime(1970, 1, 1, 0, 0, tzinfo=tzlocal())

    def _naive_is_dst(self, dt):
        timestamp = _datetime_to_timestamp(dt)
>       return time.localtime(timestamp + time.timezone).tm_isdst
E       OSError: [Errno 22] Invalid argument

C:\Users\ASU\miniconda3\envs\freqtrade\lib\site-packages\dateutil\tz\tz.py:260: OSError
```

and `rpc.test_rpc.test_rpc_health` with log:
```python
tests\rpc\test_rpc.py:1249 (test_rpc_health)
mocker = <pytest_mock.plugin.MockerFixture object at 0x00000240AB91AFE0>
default_conf = {'amend_last_stake_amount': False, 'cancel_open_orders_on_exit': False, 'candle_type_def': <CandleType.SPOT: 'spot'>, 'datadir': WindowsPath('D:/Work/freqtrade/tests/testdata'), ...}

    def test_rpc_health(mocker, default_conf) -> None:
        mocker.patch('freqtrade.rpc.telegram.Telegram', MagicMock())
    
        freqtradebot = get_patched_freqtradebot(mocker, default_conf)
        rpc = RPC(freqtradebot)
>       result = rpc._health()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = tzlocal(), dt = datetime.datetime(1970, 1, 1, 0, 0, tzinfo=tzlocal())

    def _naive_is_dst(self, dt):
        timestamp = _datetime_to_timestamp(dt)
>       return time.localtime(timestamp + time.timezone).tm_isdst
E       OSError: [Errno 22] Invalid argument

C:\Users\ASU\miniconda3\envs\freqtrade\lib\site-packages\dateutil\tz\tz.py:260: OSError
```

--- a/freqtrade/freqtradebot.py
+++ b/freqtrade/freqtradebot.py
@@ -127,7 +127,7 @@ class FreqtradeBot(LoggingMixin):
                 for minutes in [0, 15, 30, 45]:
                     t = str(time(time_slot, minutes, 2))
                     self._schedule.every().day.at(t).do(update)
-        self.last_process = datetime(1970, 1, 1, tzinfo=timezone.utc)
+        self.last_process: Optional[datetime] = None
 
         self.strategy.ft_bot_start()
         # Initialize protections AFTER bot start - otherwise parameters are not loaded.

--- a/freqtrade/rpc/api_server/api_schemas.py
+++ b/freqtrade/rpc/api_server/api_schemas.py
@@ -456,5 +456,5 @@ class SysInfo(BaseModel):
 
 
 class Health(BaseModel):
-    last_process: datetime
-    last_process_ts: int
+    last_process: Optional[datetime]
+    last_process_ts: Optional[int]

--- a/freqtrade/rpc/api_server/api_v1.py
+++ b/freqtrade/rpc/api_server/api_v1.py
@@ -346,4 +346,4 @@ def sysinfo():
 
 @router.get('/health', response_model=Health, tags=['info'])
 def health(rpc: RPC = Depends(get_rpc)):
-    return rpc._health()
+    return rpc.health()

--- a/freqtrade/rpc/rpc.py
+++ b/freqtrade/rpc/rpc.py
@@ -89,7 +89,7 @@ class RPC:
     # Bind _fiat_converter if needed
     _fiat_converter: Optional[CryptoToFiatConverter] = None
 
-    def __init__(self, freqtrade: "FreqtradeBot") -> None:
+    def __init__(self, freqtrade) -> None:
         """
         Initializes all enabled rpc modules
         :param freqtrade: Instance of a freqtrade bot

--- a/freqtrade/rpc/rpc.py
+++ b/freqtrade/rpc/rpc.py
@@ -89,7 +89,7 @@ class RPC:
     # Bind _fiat_converter if needed
     _fiat_converter: Optional[CryptoToFiatConverter] = None
 
-    def __init__(self, freqtrade) -> None:
+    def __init__(self, freqtrade: "FreqtradeBot") -> None:
         """
         Initializes all enabled rpc modules
         :param freqtrade: Instance of a freqtrade bot
@@ -1198,10 +1198,17 @@ class RPC:
             "ram_pct": psutil.virtual_memory().percent
         }
 
-    def _health(self) -> Dict[str, Union[str, int]]:
+    def health(self) -> Dict[str, Optional[Union[str, int]]]:
         last_p = self._freqtrade.last_process
+        if last_p is None:
+            return {
+                "last_process": None,
+                "last_process_loc": None,
+                "last_process_ts": None,
+            }
+
         return {
-            'last_process': str(last_p),
-            'last_process_loc': last_p.astimezone(tzlocal()).strftime(DATETIME_PRINT_FORMAT),
-            'last_process_ts': int(last_p.timestamp()),
+            "last_process": str(last_p),
+            "last_process_loc": last_p.astimezone(tzlocal()).strftime(DATETIME_PRINT_FORMAT),
+            "last_process_ts": int(last_p.timestamp()),
         }

--- a/freqtrade/rpc/telegram.py
+++ b/freqtrade/rpc/telegram.py
@@ -1527,7 +1527,7 @@ class Telegram(RPCHandler):
         Handler for /health
         Shows the last process timestamp
         """
-        health = self._rpc._health()
+        health = self._rpc.health()
         message = f"Last process: `{health['last_process_loc']}`"
         self._send_msg(message)
 

--- a/tests/rpc/test_rpc.py
+++ b/tests/rpc/test_rpc.py
@@ -1252,6 +1252,6 @@ def test_rpc_health(mocker, default_conf) -> None:
 
     freqtradebot = get_patched_freqtradebot(mocker, default_conf)
     rpc = RPC(freqtradebot)
-    result = rpc._health()
-    assert result['last_process'] == '1970-01-01 00:00:00+00:00'
-    assert result['last_process_ts'] == 0
+    result = rpc.health()
+    assert result['last_process'] is None
+    assert result['last_process_ts'] is None

--- a/tests/rpc/test_rpc_apiserver.py
+++ b/tests/rpc/test_rpc_apiserver.py
@@ -1801,8 +1801,8 @@ def test_health(botclient):
 
     assert_response(rc)
     ret = rc.json()
-    assert ret['last_process_ts'] == 0
-    assert ret['last_process'] == '1970-01-01T00:00:00+00:00'
+    assert ret["last_process_ts"] is None
+    assert ret["last_process"] is None
 
 
 def test_api_ws_subscribe(botclient, mocker):

